### PR TITLE
ecub joint setup configuration: increasing current and velocity limits

### PR DESCRIPTION
**Pull request by divyashah**
I further updated the current the velocity limits in the configuration file  under the `5-setup` subfolder for the testing of the ecub joint motor with the hysteresis dynamometer. 

The 2FOC board often reaches it's I2t limits while testing and so I would like to increase it further. 

cc @ale-git

### CODE REVIEW

**Comment by pattacini** [comment]
Hi @divyashah 

This PR is not mergeable as it contains conflicts.
Can you please sort it out?

Ideally, it suffices to fix and push a new commit; the PR will get updated.
Don't close this PR then, to start it over.

**Review by pattacini — CHANGES_REQUESTED** [review]
In particular, these are the conflicts to be resolved:

image

**Comment by divyashah** [comment]
> In particular, these are the conflicts to be resolved:
> 
> image

Thanks @pattacini for pointing it out! 
I think I might have included the previous commit by mistake which was causing the conflict. I tried to resolve it on the web. Let me know if it is mergeable now or not.

File: experimentalSetups/5-setup/hardware/motorControl/5-setup-mc.xml [hunk 1]
```diff
@@ -47,10 +47,10 @@
         <param name="outputType">             pwm                   </param>
         <param name="fbkControlUnits">        metric_units          </param>
         <param name="outputControlUnits">     machine_units         </param>
-        <param name="kp">           500              </param>
+        <param name="kp">           -500              </param>
         <param name="kd">           0                </param>
-        <param name="ki">           100              </param>
-        <param name="maxOutput">    8000           </param>
+        <param name="ki">           -100              </param>
```
Comment (pattacini) [inline experimentalSetups/5-setup/hardware/motorControl/5-setup-mc.xml:52]: @divyashah, are you sure about these negative signs?
Are they intended?

Comment (divyashah) [reply experimentalSetups/5-setup/hardware/motorControl/5-setup-mc.xml:52]: Yes, we had to change the order of the motor phase wires on the 2FOC board, and to compensate for that we had changed the gains into negative.

**Review by pattacini — APPROVED** [review]

**Comment by divyashah** [comment]
Great! Thanks @pattacini!

**Pull request merged**